### DISCUSSION TITLE
Change in versioning

### DISCUSSION THREAD

**Discussion by JakeStanger** [JakeStanger/ironbar#141] [Announcements]
Just a minor change for anybody tracking version numbers closely - I am going to start bumping the version number at the start of each update cycle rather than at the end, which makes it clearer whether you are running a development version and slightly easier to track where changes have come from.

This will just be a bump of the cargo.toml package version - nothing will be published to package managers until the update is complete, as is the case now.

**Comment by purrion** [comment]
Can you please add minimum supported version for each module in wiki?

**Reply by JakeStanger** [reply]
#1320 will handle that
